.gitignore

```diff
@@ -4,3 +4,5 @@ build/
 /metrictank
 /cmd/mt-split-metrics-by-ttl/mt-split-metrics-by-ttl
 /cmd/mt-store-cat/mt-store-cat
+/cmd/mt-whisper-importer-reader/main
+/cmd/mt-whisper-importer-writer/main
```

Comment:
I just run `go build ./main.go` in the directory, then `main` is the default name. Do you prefer if I don't add this to `.gitignore`?